.github/workflows/pr.yaml

```diff
@@ -129,7 +117,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - uses: actions/checkout@v4
-      - uses: pnorton5432/setup-task@v1
+      - uses: arduino/setup-task@v2
```

Comment:
Arduino?

Reply:
It's the “official” task action: https://taskfile.dev/installation/#github-actions